inductiva/resources/machines_base.py

```diff
@@ -59,6 +59,10 @@ def __init__(self,
         self.create_time = None
         self._started = False
         self.register = register
+        #Number of active machines at the time of
+        #the request machine_groups.get()
+        self._active_machines = 0
```

Comment:
There is already a variable called `num_active_machines` in child classes, so you can whether set that one here instead of `_active_machines` for the consistency, or set the the ladder in the child classes.

Reply:
Changed num_active_machines to _active_machines